sdk/cosmos/azure-cosmos-spark_3-1_2-12/pom.xml

```diff
@@ -6,7 +6,7 @@
   <parent>
     <groupId>com.azure.cosmos.spark</groupId>
     <artifactId>azure-cosmos-spark_3_2-12</artifactId>
-    <version>4.6.0-beta.1</version> <!-- {x-version-update;com.azure.cosmos.spark:azure-cosmos-spark_3-1_2-12;current} -->
+    <version>1.0.0</version> <!-- dummy version - will never be published -->
```

Comment:
Even though this won't be shipped, we still want to follow the versioning guidelines for this repo. Some examples are our perf and stress libraries for client libraries. We don't update the version_client.txt entry.

Reply:
I can't because the eng\scripts\generate_from_source_pom.py would then pick-it up and include it in the generated pom.xml (which caused the build break yesterday)

Reply:
Resolved this by duplicating the poms instead of trying to inheriting them - that worked well except for the conflict between version enforcement and generate_source_pom script